### PR TITLE
[NPC] Add additional dialog for Standing Bear

### PR DESCRIPTION
**Pull request by kaincenteno**
standing bear does not rotate to talk to you if you are inside the dock, instead Emyr will be the one to talk to you

**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

have tested the CSID, only need to test the script but its pretty simple

### CODE REVIEW

File: scripts/zones/Mhaura/npcs/Standing_Bear.lua [hunk 1]
```diff
@@ -10,7 +10,10 @@ entity.onTrade = function(player, npc, trade)
 end
 
 entity.onTrigger = function(player, npc)
-    player:startEvent(14)
+    if player:getZPos() > 38.5 then
+        player:startEvent(14)
+    else
+        player:startEvent(235)
```
Comment (Xaver-DaRed) [inline scripts/zones/Mhaura/npcs/Standing_Bear.lua:16]: Missing an `end` after this line.

**Review by Xaver-DaRed — CHANGES_REQUESTED** [review]
Other than that, this seem correct.
imagen

File: scripts/zones/Mhaura/npcs/Standing_Bear.lua (hunk 1 above)
Comment (kaincenteno) [reply scripts/zones/Mhaura/npcs/Standing_Bear.lua:16]: i knew i was forgetting something than you :)

**Review by Xaver-DaRed — APPROVED** [review]

**Review by claywar — APPROVED** [review]

**Comment by kaincenteno** [comment]
Tested, working as expected :)

**Pull request merged**